src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/CCDConfigConstant.java

```diff
@@ -445,6 +445,8 @@ public class CCDConfigConstant {
     public static final String RESP_CORRESPONDENCE_COLLECTION = "respCorrespondenceDocsColl";
     public static final String RESP_OTHER_COLLECTION = "respOtherCollection";
 
+    // Manage case documents
```

Comment:
Remove the comment here as its not needed

Reply:
Comment removed